astore/server/auth/factory.go

```diff
@@ -24,6 +29,8 @@ func DefaultFlags() *Flags {
 
 func (f *Flags) Register(set kflags.FlagSet, prefix string) *Flags {
 	set.DurationVar(&f.TimeLimit, prefix+"time-limit", f.TimeLimit, "How long to wait at most for the user to complete authentication, before freeing resources")
+	set.StringVar(&f.Principals, prefix+"principals", f.Principals, "Allowed principals and the ability to auth,comma separated string")
+	set.StringVar(&f.PathToCA, prefix+"ca", f.PathToCA, "Path to the certificate authority private file")
```

Comment:
For files, use `set.ByteFileVar`, defined in `lib/kflags/flags.go`. By identifying the variable as the path to a file (and not just as a string), the file will be loadable / embeddable correctly, and flag parsing can take care of checking if the file is loadable.